### DISCUSSION TITLE
SunbirdED - Portal :: How to restrict popup (Are you sure, you want to leave this page?) while copying the question set

### DISCUSSION THREAD

**Discussion by princegupta1131** [Sunbird-inQuiry/Community#88] [General Discussions]
**In Sunbird Ed - Portal** - on click of copy Question set. getting the below popup with the msg "Are you sure, you want to leave this page?"  instead, without showing this popup want to redirect to the question set editor.

![image](https://github.com/Sunbird-inQuiry/Community/assets/114015020/f1cb2707-341a-4edb-a28b-55cfc685c9ac)

can anyone please suggest how we can restrict this popup only while copying the question set?

CC: @sajeshkayyath @krgauraw  @rajeevsathish @vinukumar-vs  @Krishnaj20 @PoojaJManjooran @kameshbhr @surendrasinghs @rajnishdargan @vaibhavbhuva @pallakartheekreddy

**Comment by sajeshkayyath** [comment]
@princegupta1131 - this seems to be external to inQuiry component, hence we will not have any way to control it.

In general context, what I see from references is that you might have to look at the `onbeforeunload` events to see if this can be achieved.
This may need to be further explored with ED - @vinukumar-vs 

@krgauraw @rajeevsathish @vinukumar-vs @Krishnaj20 @PoojaJManjooran @kameshbhr @surendrasinghs @rajnishdargan @vaibhavbhuva @pallakartheekreddy
